utils/db/repos.ts

```diff
@@ -0,0 +1,15 @@
+import conn from '.';
+import type { RepoIdentifier } from '../../types/repository';
+import { convert } from './converter';
+
+export const setRepoConfig = async (repo: RepoIdentifier, configType: 'auto_assign' | 'comment', value: boolean) => {
```

Comment:
everytime we add a new type of setting, will we have to change the type from 'auto_assign' | 'comment' ? We might have a lot of settings, which could be more complex (setting inside a sub-setting etc.)

Reply:
Whenever that happens (a new setting is added), a lot of work will need to be done. In my opinion, a statically defined type must be modified for that. 

And if we introduce more complex settings (sub settings etc.) we will have to modify the entire UI as well. 

If this type is used in many places, we can make a dedicated TS type for it. I am not sure what else can be done to improve this. But if you have any recommendations, please let me know.

Reply:
Cool, we'll burn that bridge when we have to cross it